src/util/workers.h

```diff
@@ -13,18 +13,12 @@
 #include "config.h"
 #include <assert.h>
 
-#define USE_BURST_THREADS() (RSGlobalConfig.numWorkerThreads && RSGlobalConfig.mt_mode == MT_MODE_ONLY_ON_OPERATIONS)
-
 // create workers thread pool
 // returns REDISMODULE_OK if thread pool created, REDISMODULE_ERR otherwise
 int workersThreadPool_CreatePool(size_t worker_count);
 
-/** Set the number of workers.
- * If @param worker_count is 0, the current living workers will continue to execute pending jobs and then terminate.
- * No new jobs should be added after setting the number of workers to 0.
- * @returns the number of workers ready to accept new jobs after the change.
-*/
-size_t workersThreadPool_SetNumWorkers(size_t worker_count);
+// Set the number of workers according to the configuration.
```

Comment:
Is there an appropriate location for the deleted comment?

Reply:
I can add a description on `workersThreadPool_SetNumWorkers` on the c file, and also explain better what configurations and states affect the function decision